atomicwrites/__init__.py

```diff
@@ -54,6 +60,9 @@ def replace_atomic(src, dst):
     Move ``src`` to ``dst``. If ``dst`` exists, it will be silently
     overwritten.
 
+    On POSIX this will also open the parent directory of ``dst`` and execute
+    ``fsync`` on it to write the new filename to disk.
+
```

Comment:
I'm a bit unsure about having a function called `replace_atomic` that is executing more than one atomic call, but since this seems like an easy oversight it might be better API design for the avg user.